grc/compiler.py

```diff
@@ -56,6 +58,9 @@ def main(args=None):
         os.path.exists(grc_file) or exit('Error: missing ' + grc_file)
         Messages.send('\n')
 
+        if args.trusted:
+            platform.trusted_flowgraphs.add(str(grc_file))
```

Comment:
I suspect this fails for flowgraphs with dependent hier_block in them. These can get loaded and generated as well, but they won't be trusted by default. I would change this to simply trust all flowgraphs.